### DISCUSSION TITLE
Petlink Transfer

### DISCUSSION THREAD

**Discussion by fopashelter** [sheltermanager/asm3#1511] [General]
Should  microchip information automatically transfer from ASM to Petlink when an animal is adopted in the system?  We are unclear on the steps needed to ensure that the microchip information entered in the system transfers to Petlink.

**Comment by Tetzor** [comment]
Hi, You need enter your Petlink login details in Publishing->Set publishing options->Petlink and check the enabled box. The links below contain more information on Petlink and how to select when microchips should be registered.

https://sheltermanager.com/repo/asm3_help/publishing.html#petlink
https://sheltermanager.com/repo/asm3_help/publishing.html#all-publishers
